src/components/input/radio/button/style.css

```diff
@@ -0,0 +1,27 @@
+:host {
+	display: flex;
+	margin-right: 1em;
```

Comment:
remove this and replace it with `gap` on the parent. (i.e. on the `smoothly-input-radio`)

Reply:
also consider using `space-between `instead of `gap`. Not entirely sure `space-between` is better but try it.

Reply:
you mean justify-content: space-bteween? that spreads em the full width and that looks wierd if the element takes the full width of the screen: 
![image](https://github.com/utily/smoothly/assets/70133569/8152ffef-336b-4cd8-8815-b1608ec656e3)
gap is the one!